app/forms/forms/submission_email_form.rb

```diff
@@ -0,0 +1,94 @@
+class Forms::SubmissionEmailForm
+  include ActiveModel::Model
+  include ActiveModel::Validations
+
+  attr_accessor :form, :temporary_submission_email, :email_code, :confirmation_code, :current_user
+
+  EMAIL_REGEX = /.*@.*/
+  GOVUK_EMAIL_REGEX = /\.gov\.uk\z/i
+
+  validates :temporary_submission_email, presence: true
```

Comment:
In the existing email form, the `presence` validation has an `if: -> { form.live? }` condition on it so that the user can put an empty value in there if the form is in draft. Is that behaviour being intentionally removed?